src/test/java/io/confluent/idesidecar/restapi/auth/RefreshCCloudTokensBeanTest.java

```diff
@@ -28,6 +29,7 @@ class RefreshCCloudTokensBeanTest {
   RefreshCCloudTokensBean refreshCCloudTokensBean;
 
   @Test
+  @RetryingTest(5)
```

Comment:
I added the `RetryingTest` annotation because I noticed that this test randomly fails from time to time. We should try to figure out the underlying issue once time permits.